lib/gds_api/json_client.rb

```diff
@@ -68,18 +68,6 @@ def delete_json(url, params = {}, additional_headers = {})
       do_json_request(:delete, url, params, additional_headers)
     end
 
-    def delete_json_with_params!(url, params, additional_headers = {})
```

Comment:
If you're removing this method then this delegation needs to be cleaned up - https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/base.rb#L26

Reply:
Ah, good spot - removed too :-).